client/src/components/feddback-comment.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap'
+import '../style/feedback.css'
+
+class FeedbackComment extends Component {
+
+
+    render() {
+
+        return (
+
+            <Container className="feedbackUserContainer">
+                <img src="/img/defAvatar.png" width="50px" height="50px" alt="defAvava"></img>
+                <div className="feedbackMess">
+                    <strong className="nameDate">{this.props.item.userName}</strong>
+                    <small className="nameDate">{this.props.item.date}</small>
+                    <hr className="hrStyle" />
+                    <div>{this.props.item.description}</div>
+                    <hr className="hrStyle" />
+                    <div>
+
+                        <span className="answerToFeedback" title="answer to feedback">answer to</span>
```

Comment:
Make className simple. Call it just "answer", or "response"

Reply:
ok